### PR TITLE
chore: cleanup all text colors

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Refactor all text colors to use new naming conventions.
- (Lots of changes as they touch about everything)

🚨 Be aware with open PR's as they might re-introduce colors that no longer work now 🚨

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://chore-cleanup-text-colors.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/tailwind.config.ts [hunk 1]
```diff
@@ -54,25 +54,11 @@ export default {
         bg: {
           bun: 'var(--theme-background-bun)',
           onion: 'var(--theme-background-onion)',
-          pepper: 'var(--theme-background-pepper)',
```
Comment (nensidosari) [inline packages/shared/tailwind.config.ts:57]: This is not in our guidelines anymore?

**Review by nensidosari — COMMENTED** [review]
🎉

few comments just for check

**Review by nensidosari — APPROVED** [review]

File: packages/extension/src/companion/CompanionEngagements.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export function CompanionEngagements({
 
   return (
     <div
-      className="flex items-center gap-x-4 py-1 text-theme-label-tertiary typo-callout"
+      className="flex items-center gap-x-4 py-1 text-text-tertiary typo-callout"
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/extension/src/companion/CompanionEngagements.tsx:39]: Hehe yeah we got few like that double naming, but matches design spec as we decided 🙏
It also makes it more explicit technically they should not be used as bg-text-color for instance 🙏

File: packages/shared/src/components/profile/common.tsx [hunk 3]
```diff
@@ -25,7 +22,7 @@ export function CommentContent({
       {...props}
       className={classNames(
         className,
-        'theme-label-primary multi-truncate m-0 whitespace-pre-wrap break-words p-0 typo-callout tablet:mr-6 tablet:flex-1',
+        'multi-truncate m-0 whitespace-pre-wrap break-words p-0 typo-callout tablet:mr-6 tablet:flex-1',
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/profile/common.tsx:25]: Yeah it was already a half one and unused anyway

**Pull request merged**